crates/aptos/CHANGELOG.md

```diff
@@ -7,6 +7,10 @@ All notable changes to the Aptos CLI will be captured in this file. This project
 - Added an option `--print-metadata` to the command `aptos move download` to print out the metadata of the package to be downloaded.
   - Example: `aptos move download  --account 0x1 --package AptosFramework --url https://mainnet.aptoslabs.com/v1 --print-metadata`
 
+### Updated
+- The `--with-faucet` flag has been removed from `aptos node run-local-testnet`, we now run a faucet by default. To disable the faucet use the `--no-faucet` flag.
```

Comment:
what happens if we still use `--with-faucet`? will it explode?

Reply:
Nope, it's actually still there, I just hid it. Otherwise yeah things would blow up.